lib/workers/repository/process/limits.ts

```diff
@@ -10,63 +11,98 @@ export async function getPrHourlyRemaining(
 ): Promise<number> {
   if (config.prHourlyLimit) {
     logger.debug('Calculating hourly PRs remaining');
-    const prList = await platform.getPrList();
-    const currentHourStart = moment({
-      hour: moment().hour(),
-    });
-    logger.debug(`currentHourStart=${String(currentHourStart)}`);
     try {
+      const prList = await platform.getPrList();
+      const currentHourStart = moment({
+        hour: moment().hour(),
+      });
+      logger.debug(`currentHourStart=${String(currentHourStart)}`);
       const soFarThisHour = prList.filter(
         (pr) =>
           pr.branchName !== config.onboardingBranch &&
           moment(pr.createdAt).isAfter(currentHourStart)
       );
-      const prsRemaining = config.prHourlyLimit - soFarThisHour.length;
+      const prsRemaining = Math.max(
+        0,
+        config.prHourlyLimit - soFarThisHour.length
+      );
       logger.debug(`PR hourly limit remaining: ${prsRemaining}`);
-      // istanbul ignore if
-      if (prsRemaining <= 0) {
-        logger.debug(
-          {
-            prs: prsRemaining,
-          },
-          'Creation of new PRs is blocked by existing PRs'
-        );
-      }
       return prsRemaining;
     } catch (err) {
       logger.error('Error checking PRs created per hour');
+      return config.prHourlyLimit;
+    }
+  }
+  return 99;
+}
+
+export async function getConcurrentPrsRemaining(
+  config: RenovateConfig
+): Promise<number> {
+  if (config.prConcurrentLimit) {
+    logger.debug(`Calculating prConcurrentLimit (${config.prConcurrentLimit})`);
+    try {
+      const prList = await platform.getPrList();
+      const openPrs = prList.filter(
+        (pr) =>
+          pr.state === PrState.Open && pr.branchName !== config.onboardingBranch
+      );
```

Comment:
From the [GitHub implementation](https://github.com/renovatebot/renovate/blob/73f11c09ee7a0c5b2952e028db6bb22c75b312d7/lib/platform/github/index.ts#L702-L742), it seems all PRs get returned by `getPrList`. Should this be filtered to PRs that match the Renovate `branchName`?

Reply:
The platform layer could possibly sort by the PR author. Alternatively we could filter here based on `branchPrefix`.